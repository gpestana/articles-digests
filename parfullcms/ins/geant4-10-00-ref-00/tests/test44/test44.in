#================================================
#     Macro file for test44
#     11.03.2008 A.Bagulya
#================================================
/control/verbose 2
/run/verbose 1
/tracking/verbose 1
#
/testhadr/TargetMat        G4_Al 
/testhadr/TargetRadius     5  cm
/testhadr/TargetLength     4  cm
/testhadr/NumberDivZ       40
/testhadr/PrintModulo      10
#
#/testhadr/CutsAll          1 mm
#/testhadr/Physics          QBBC
#/testhadr/Physics          emstandard_opt2
#
/run/initialize
/process/eLoss/subsec true
#
/gun/particle proton
/gun/energy 100. MeV
/run/beamOn 1
#
