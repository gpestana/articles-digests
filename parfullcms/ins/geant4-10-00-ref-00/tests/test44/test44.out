
        ############################################
        !!! WARNING - FPE detection is activated !!!
        ############################################

*************************************************************
 Geant4 version Name: geant4-10-00-ref-00    (6-December-2013)
                      Copyright : Geant4 Collaboration
                      Reference : NIM A 506 (2003), 250-303
                            WWW : http://cern.ch/geant4
*************************************************************

/run/verbose 1
/tracking/verbose 1
#
/testhadr/TargetMat G4_Al
/testhadr/TargetRadius 5 cm
/run/geometryModified
/testhadr/TargetLength 4 cm
/testhadr/NumberDivZ 40
/testhadr/PrintModulo 10
#
#/testhadr/CutsAll          1 mm
#/testhadr/Physics          QBBC
#/testhadr/Physics          emstandard_opt2
#
/run/initialize
### Target consist of 40 of G4_Al disks with R(mm)= 50  Width(mm)= 1  Total Length(mm)= 40  ###

***** Table : Nb of materials = 2 *****

 Material:    G4_Al    density:  2.699 g/cm3   RadL:   8.896 cm   Nucl.Int.Length:  38.877 cm 
                       Imean: 166.000 eV 

   --->  Element: Al (Al)   Z = 13.0   N =  27.0   A =  26.98 g/mole
         --->  Isotope:  Al27   Z = 13   N =  27   A =  26.98 g/mole   abundance: 100.00 %
          ElmMassFraction: 100.00 %  ElmAbundance 100.00 % 


 Material: G4_Galactic    density:  0.000 kg/m3   RadL: 204310101.835 pc   Nucl.Int.Length: 114319208.901 pc 
                       Imean:  21.800 eV   temperature:   2.73 K  pressure:   0.00 atm

   --->  Element: H (H)   Z =  1.0   N =   1.0   A =   1.01 g/mole
         --->  Isotope:    H1   Z =  1   N =   1   A =   1.01 g/mole   abundance:  99.99 %
         --->  Isotope:    H2   Z =  1   N =   2   A =   2.01 g/mole   abundance:   0.01 %
          ElmMassFraction: 100.00 %  ElmAbundance 100.00 % 



PhysicsList::SetCuts:CutLength : 700 um 
/process/eLoss/subsec true
/run/physicsModified
#
/gun/particle proton
/gun/energy 100. MeV
/run/beamOn 1

========= Table of registered couples ==============================

Index : 0     used in the geometry : Yes
 Material : G4_Galactic
 Range cuts        :  gamma  700 um     e-  700 um     e+  700 um  proton 700 um 
 Energy thresholds :  gamma  990 eV     e-  990 eV     e+  990 eV  proton 70 keV
 Region(s) which use this couple : 
    DefaultRegionForTheWorld

Index : 1     used in the geometry : Yes
 Material : G4_Al
 Range cuts        :  gamma  700 um     e-  700 um     e+  700 um  proton 700 um 
 Energy thresholds :  gamma  5.85564 keV    e-  460.395 keV    e+  442.201 keV proton 70 keV
 Region(s) which use this couple : 
    DefaultRegionForTheWorld

====================================================================

### Run 0 starts.
### Run 0 start
### Histo books 1 histograms in <test44.root>
EventAction: Event # 0 started

*********************************************************************************************************
* G4Track Information:   Particle = proton,   Track ID = 1,   Parent ID = 0
*********************************************************************************************************

Step#    X(mm)    Y(mm)    Z(mm) KinE(MeV)  dE(MeV) StepLeng TrackLeng  NextVolume ProcName
    0        0        0      -20       100        0        0         0      Target initStep
    1        0        0      -19      98.5     1.48        1         1      Target Transportation
    2 -0.00254 -0.00818      -18      97.2     1.36        1         2      Target Transportation
    3 -0.00402  -0.0153      -17      95.8     1.39        1         3      Target Transportation
    4  -0.0576   -0.107      -16      94.2     1.54     1.01      4.01      Target Transportation
    5   -0.111   -0.193      -15      92.9     1.37     1.01      5.01      Target Transportation
    6   -0.169   -0.283      -14      91.4     1.46     1.01      6.02      Target Transportation
    7    -0.24   -0.368      -13      89.9     1.53     1.01      7.02      Target Transportation
    8   -0.315   -0.453      -12      88.5     1.39     1.01      8.03      Target Transportation
    9   -0.393   -0.534      -11      86.9     1.57     1.01      9.04      Target Transportation
   10   -0.467   -0.614      -10      85.3     1.64     1.01        10      Target Transportation
   11   -0.544   -0.688       -9      83.7     1.62     1.01        11      Target Transportation
   12   -0.607   -0.756       -8      82.1     1.53        1      12.1      Target Transportation
   13   -0.673   -0.823       -7      80.3     1.78        1      13.1      Target Transportation
   14   -0.747   -0.883       -6      78.7     1.62        1      14.1      Target Transportation
   15    -0.84    -0.95       -5        77     1.69     1.01      15.1      Target Transportation
   16   -0.918    -1.02       -4      75.2      1.8     1.01      16.1      Target Transportation
   17   -0.997     -1.1       -3      73.6     1.69     1.01      17.1      Target Transportation
   18    -1.09    -1.19       -2      71.8     1.77     1.01      18.1      Target Transportation
   19    -1.17    -1.28       -1      69.9      1.9     1.01      19.1      Target Transportation
   20    -1.26    -1.37        0        68     1.86     1.01      20.1      Target Transportation
   21    -1.34    -1.44        1      66.2      1.8     1.01      21.1      Target Transportation
   22    -1.41    -1.51        2      64.2     2.03     1.01      22.1      Target Transportation
   23    -1.48    -1.57        3      62.1     2.08        1      23.1      Target Transportation
   24    -1.56    -1.63        4      60.1        2     1.01      24.1      Target Transportation
   25    -1.63    -1.71        5      58.2     1.96     1.01      25.1      Target Transportation
   26     -1.7    -1.78        6        56      2.2     1.01      26.1      Target Transportation
   27    -1.79    -1.85        7      53.7     2.27     1.01      27.1      Target Transportation
   28    -1.87    -1.92        8      51.4     2.31     1.01      28.1      Target Transportation
   29    -1.94       -2        9      48.9     2.46     1.01      29.2      Target Transportation
   30    -2.01    -2.06       10      46.5      2.4        1      30.2      Target Transportation
   31    -2.06    -2.13       11        44     2.55        1      31.2      Target Transportation
   32    -2.13     -2.2       12      40.9     3.02     1.01      32.2      Target Transportation
   33     -2.2     -2.3       13      37.8     3.16     1.01      33.2      Target Transportation
   34    -2.29    -2.38       14      34.5     3.23     1.01      34.2      Target Transportation
   35    -2.38    -2.45       15      31.3     3.22     1.01      35.2      Target Transportation
   36     -2.5    -2.49       16      27.6     3.78     1.01      36.2      Target Transportation
   37    -2.54    -2.58     16.9      23.6     3.94    0.915      37.1      Target hIoni
   38    -2.55    -2.58       17      23.2     0.39   0.0906      37.2      Target Transportation
   39     -2.6    -2.62     17.7      19.7     3.51    0.717      37.9      Target hIoni
   40    -2.61    -2.64       18      18.1     1.59    0.288      38.2      Target Transportation
   41    -2.64    -2.69     18.5      15.1     3.06    0.517      38.7      Target hIoni
   42    -2.65    -2.74     18.9      12.1     2.98    0.414      39.1      Target hIoni
   43    -2.65    -2.74       19      11.5    0.568   0.0762      39.2      Target Transportation
   44    -2.67    -2.78     19.3      8.65     2.85    0.312      39.5      Target hIoni
   45    -2.68    -2.79     19.6      5.77     2.88    0.242      39.8      Target hIoni
   46    -2.67    -2.82     19.7       2.6     3.17    0.177      39.9      Target hIoni
   47    -2.67    -2.83     19.8         0      2.6   0.0649        40      Target hIoni
Run terminated.
Run Summary
  Number of events processed : 1
  User=0.01s Real=0s Sys=0s
RunAction: End of run actions are started
HistoManager: End of run actions are started
========================================================
Beam particle                        proton
Beam Energy(MeV)                     100
Number of events                     1
Average energy deposit (MeV)         100   RMS(MeV) 0
Average number of steps of primary   47
========================================================

#
G4 kernel has come to Quit state.
================== Deleting memory pools ===================
Number of memory pools allocated: 11 of which, static: 0
Dynamic pools deleted: 11 / Total memory freed: 0.27 Mb
============================================================
RunManagerKernel is deleted. Good bye :)
